patches/api/0300-Missing-Entity-API.patch

```diff
@@ -76,6 +76,60 @@ index 0000000000000000000000000000000000000000..39ad7d283609d7e427a2ab35b6fad839
 +    SchoolableFish getSchoolLeader();
 +
 +}
+diff --git a/src/main/java/io/papermc/paper/potion/SuspiciousEffectEntry.java b/src/main/java/io/papermc/paper/potion/SuspiciousEffectEntry.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..d2e1b9d867394e46e9491bf9e9cfa53c843ef16b
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/potion/SuspiciousEffectEntry.java
+@@ -0,0 +1,36 @@
++package io.papermc.paper.potion;
++
++import org.bukkit.potion.PotionEffectType;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Represents an effect paired with a duration.
++ */
++public sealed interface SuspiciousEffectEntry permits SuspiciousEffectEntryImpl {
++
++    /**
++      * Gets the effect type.
++     *
++      * @return type
++      */
++    @NotNull PotionEffectType effect();
++
++    /**
++      * Gets the duration for this effect instance.
++      *
++      * @return duration in ticks
++      */
++    int duration();
++
++    /**
++     * Gets a new instance of PotionEffectPair.
++     *
++     * @param effectType effect type
++     * @param duration duration
++     * @return new instance
++     */
++    @NotNull
++    static SuspiciousEffectEntry suspiciousEffectEntry(PotionEffectType effectType, int duration) {
```

Comment:
So I know I said to name it this for static imports... but I feel like I changed my own mind. We don't do that anywhere else in the API really for static factory-type methods. Do we want to here? Its just kinda long instead of `.create` or something.